### PR TITLE
email feed

### PR DESCRIPTION
**Pull request by joshm1**
this is still a work in progress, but I'm closing in on sending these emails on a cron.  any early reviewers?

@eishay @andrewconner @stkem

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala [hunk 1]
```diff
@@ -353,6 +354,7 @@ object Curator extends Service {
   object internal {
     def adHocRecos(userId: Id[User], n: Int) = ServiceRoute(POST, "/internal/curator/adHocRecos", Param("userId", userId), Param("n", n))
     def updateUriRecommendationFeedback(userId: Id[User], uriId: Id[NormalizedURI]) = ServiceRoute(POST, "/internal/curator/updateUriRecommendationFeedback", Param("userId", userId), Param("uriId", uriId))
+    def triggerEmail(code: String) = ServiceRoute(POST, "/internal/curator/triggerEmail", Param("code", code))
```
Comment (joshm1) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala:359]: this endpoint isn't working. not sure why yet

File: kifi-backend/modules/curator/app/com/keepit/curator/commanders/email/EngagementEmailActor.scala [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import com.keepit.common.akka.FortyTwoActor
+import com.keepit.common.healthcheck.AirbrakeNotifier
+import com.keepit.common.logging.Logging
+import com.keepit.curator.commanders.RecommendationGenerationCommander
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+
+case class EngagementEmailType(message: String) extends AnyVal
+
+object EngagementEmailTypes {
+  val FEED = EngagementEmailType("feed")
```
Comment (eishay) [inline kifi-backend/modules/curator/app/com/keepit/curator/commanders/email/EngagementEmailActor.scala]: if you don't care about the internals, I would do 

``` scala
object EngagementEmailTypes {
  object FEED
}
```

so you know its a singleton and you can match on it

File: kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala (hunk 1 above)
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala:359]: Specifically what's breaking?

Comment (joshm1) [inline kifi-backend/modules/common/app/com/keepit/common/routes/Router.scala:359]: i figured it out (with @leogrim's help).  Curator routes were loaded into kifi-backend, so it was throwing a NO HANDLER error.

File: kifi-backend/modules/common/test/com/keepit/shoebox/FakeShoeboxServiceClientImpl.scala [hunk 3]
```diff
@@ -536,6 +536,11 @@ class FakeShoeboxServiceClientImpl(val airbrakeNotifier: AirbrakeNotifier) exten
     Future.successful(allProbabilisticExperimentGenerators.values.filter(_.isActive).toSeq)
   }
 
+  def getUsersByExperiment(experimentType: ExperimentType): Future[Set[User]] = {
+    println("allUsers:" + allUsers.map(_._2))
```
Comment (andrewconner) [inline kifi-backend/modules/common/test/com/keepit/shoebox/FakeShoeboxServiceClientImpl.scala]: Intended?

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala [hunk 4]
```diff
@@ -195,6 +199,20 @@ class HomeController @Inject() (
     }
   }
 
+  // this is for testing and will eventually be thrown away
+  def kifeeeedEmail(code: String) = HtmlAction.authenticatedAsync { request =>
```
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala:203]: You can hit the endpoint on curator directly (via curl, from marvin or any aws machine) so you don't need to so this sort of thing for testing.

**Pull request merged**